### DISCUSSION TITLE
Choppy video when streamming over WiFi

### DISCUSSION THREAD

**Discussion by lailamor** [scivision/PyLivestream#100] [Q&A]
### Description
When streaming over WiFi, the video stream gets choppy. Audio seems OK though.

### Reproduce
If connected to Ethernet, disconnect
Connect to WiFi
Stream to twitch.tv

### pylivestram Configuration
Default. Only added stream key.

### Software
* Operating system: Arch Linux
* FFmpeg 4.3.1

### Other comments
Same issue with my personal ffmpeg script. So maybe something that could be worked around rtmp?

**Comment by scivision** [comment]
This is [not unexpected](https://github.com/scivision/PyLivestream/blob/master/Troubleshooting.md) with WiFi. A lot of WiFi networks experience congestion especially in the pandemic with so many home offices. This congestion isn't noticed while watching (consuming) video due to more extensive buffering.

**Comment by lailamor** [comment]
I've tested with OBS studio. With OBS this issue doesn't happen... Only with ffmpeg/pylivestream. Seems OBS devs might have workarounded it or something.
